composer.json

```diff
@@ -34,7 +34,7 @@
         "wpackagist-plugin/fast-user-switching": "*",
         "wpackagist-plugin/wp-browser-update": "*",
         "relevanssi/relevanssi-premium": "*",
-        "wpackagist-plugin/google-analytics-dashboard-for-wp": "^5.3",
+        "wpackagist-plugin/google-analytics-dashboard-for-wp": "^5.3.10",
```

Comment:
The ^ symbol at the start will auto-load the .10 version without having to implicitly define it. Have you found an incompatibility with a newer version?